### ISSUE TITLE
🐛 BUG: Attribute spread breaks prettier plugin

### ISSUE THREAD

**Issue by innermatrix** [withastro/prettier-plugin-astro#70]
### Describe the Bug

Error processing a file with an attribute spread:

```
$ ./node_modules/.bin/prettier --loglevel debug --no-editorconfig --parser astro test.astro                               
[debug] normalized argv: {"_":["test.astro"],"color":true,"editorconfig":false,"loglevel":"debug","parser":"astro","config-precedence":"cli-override","debug-repeat":0,"ignore-path":".prettierignore","plugin":[],"plugin-search-dir":[]}
[debug] resolve config from 'test.astro'
[debug] loaded options `{"tabWidth":2,"useTabs":false,"printWidth":160}`
[debug] applied config-precedence (cli-override): {"parser":"astro","printWidth":160,"tabWidth":2,"useTabs":false}
test.astro[error] test.astro: Error: Unhandled node type "Spread"!
[error]     at Object.print (/[redacted]/node_modules/prettier-plugin-astro/dist/index.js:712:19)
[error]     at callPluginPrintFunction (/[redacted]/node_modules/prettier/index.js:13787:21)
[error]     at mainPrintInternal (/[redacted]/node_modules/prettier/index.js:13725:17)
[error]     at mainPrint (/[redacted]/node_modules/prettier/index.js:13707:14)
[error]     at /[redacted]/node_modules/prettier/index.js:13495:23
[error]     at AstPath.each (/[redacted]/node_modules/prettier/index.js:13482:7)
[error]     at AstPath.map (/[redacted]/node_modules/prettier/index.js:13494:10)
[error]     at Object.print (/[redacted]/node_modules/prettier-plugin-astro/dist/index.js:565:37)
[error]     at callPluginPrintFunction (/[redacted]/node_modules/prettier/index.js:13787:21)
[error]     at mainPrintInternal (/[redacted]/node_modules/prettier/index.js:13725:17)
```

with `test.astro` as follows:

```
<div {...Astro.props}><div /></div>
```

```
├── prettier-plugin-astro@0.0.10
└── prettier@2.4.1
```

### Steps to Reproduce

See above

**Comment by antonyfaris** [comment]
This will be fixed in the next release. See #66

**Comment by innermatrix** [comment]
Resolved in 0.0.11
